core/modules/commands/makelibrary.js

```diff
@@ -54,6 +54,22 @@ Command.prototype.execute = function() {
 	collectPublisherPlugins(path.resolve($tw.boot.corePath,$tw.config.pluginsPath));
 	collectPublisherPlugins(path.resolve($tw.boot.corePath,$tw.config.themesPath));
 	collectPlugins(path.resolve($tw.boot.corePath,$tw.config.languagesPath));
+	// Adding local wiki paths
+	try {
+		collectPlugins(path.resolve($tw.boot.wikiPath,$tw.config.wikiPluginsSubDir));
+	} catch (ex) {
+		console.log("No Local Plugins Folder");
```

Comment:
I see how you may used that for testing, but do we need any such error catching in the final code?